Tools/autotest/common.py

```diff
@@ -1263,9 +1252,9 @@ def check_test_syntax(self, test_file):
                 # check for lambda: test_function without paranthesis
                 faulty_strings = re.findall(r"lambda\s*:\s*\w+.\w+\s*\)", f.read())
                 if faulty_strings:
-                    self.progress("Syntax error in autotest lamda at : ")
-                    print(faulty_strings)
-                    raise ErrorException()
+                    desc = ("Syntax error in autotest lamda at : " +
```

Comment:
s/lamda/lambda

Reply:
Pre-existing! :-)  And we wouldn't want to mix in unrelated changes ;-)